protected/views/search/index.php

```diff
@@ -1,3 +1,4 @@
+<!-- Deprecated? -->
 <div class="content">
             <section class="image-background" style="background-image: url(../images/banner2.jpg);">
```

Comment:
**issue:** The url path is incorrect which should be `../../../images/new_interface_image/banner2.jpg`.

Reply:
@kencho51  you are right in that this is redundant so I removed it, however keep in mind that this page isn't really used (I think) as there is a /new search page

anyway I took the chance to update the image to the one you suggest and tweak the display properties, as it looks better that way

Reply:
@luistoptal 
1. Regarding the banner image, I have compared these two images and I have no idea which one is better:
![image](https://github.com/gigascience/gigadb-website/assets/64770635/4966a371-3fdd-4814-947f-fb39b5cca7cb)
May be @rija and @pli888  can tell which one should be used.

2. > you are right in that this is redundant so I removed it, however keep in mind that this page isn't really used (I think) as there is a /new search page

I share the same thought as yours.